pkg/resources/testdata/TestAcc_GrantApplicationRole/application/manifest.yml

```diff
@@ -0,0 +1,5 @@
+manifest_version: 1 # required
```

Comment:
is separate manifest and setup.sql needed? (we already have them in tests; check comment in test.tf file)

Reply:
Merged into one with accountRole.